### PR TITLE
feat(inventory): add tags to quick inventory

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Add tags to quick inventory outputs and refactor logic.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/providers/aws/services/s3/s3_service.py [hunk 1]
```diff
@@ -262,9 +262,12 @@ def __get_bucket_tagging__(self, bucket):
                 "TagSet"
             ]
             bucket.tags = bucket_tags
-        except ClientError as e:
-            if e.response["Error"]["Code"] == "NoSuchTagSet":
-                bucket.tags = []
+        except ClientError as error:
+            bucket_tags = []
```
Comment (jfagoagas) [inline prowler/providers/aws/services/s3/s3_service.py]: I think you need to assign this to the `bucket` object right?

**Review by jfagoagas — CHANGES_REQUESTED** [review]
Love the refactor!

File: prowler/providers/aws/services/s3/s3_service.py (hunk 1 above)
Comment (MrCloudSec) [reply prowler/providers/aws/services/s3/s3_service.py]: Good catch!!!! Thank you

Comment (MrCloudSec) [inline prowler/providers/aws/services/s3/s3_service.py]:
```suggestion
            bucket.tags = []
```

**Review by jfagoagas — APPROVED** [review]

**Pull request merged**